src/views/AnalyzePhenotypes.vue

```diff
@@ -339,6 +352,32 @@
         <div class="col-1"/>
       </div>
     </div>
+    <b-modal size="xl" id="help-one" title="Creating Starting Profile" ok-only>
+      <p>To begin, you can create a profile by pasting a list of phenotypes (<a href="https://hpo.jax.org" target="_blank">Human Phenotype Ontology</a>) and confirming your profile.</p>
+      <div class="text-center">
+        <img src="../assets/img/first-comparison.gif">
+      </div>
+      <br>
+      <p>As an alternative you can build a profile of phenotypes by choosing a gene or disease. The associated phenotypes to those entities will 
+        be populated for you.</p>
+      <div class="text-center">
+        <img src="../assets/img/first-comparison2.gif">
+      </div>
+    </b-modal>
+    <b-modal size="xl" id="help-two" title="Creating Comparison Profile" ok-only>
+      <h5>The next step is to choose what to compare your profile to. </h5>
+      <p>You may choose a blanket search by selecting all genes by a species, 
```

Comment:
Update content in lines 369 and 370 to read (updates emphasized with italics):  "You may choose _to conduct_ a blanket search by selecting _either_ all genes _from_ a single species" or all human _diseases. Once you make a selection, you may_ run the analysis."